src/sdk/lsf-sdk.js

```diff
@@ -264,10 +264,10 @@ export class LSFWrapper {
       }
 
       const newTask = await this.withinLoadingState(async () => {
-        if (!isDefined(_taskId) && !isPrevious) {
-          return tasks.loadNextTask();
-        } else {
+        if (isDefined(_taskId) || isPrevious) {
```

Comment:
I was going to ask, if the `_taskId` is not defined and `isPrevious` is `true`, is it the intention to call `loadNextTask`?

Reply:
As right now, you would be calling loadTask on `undefined` or `null` possibly.